File: mpp/LabMPPCurse/Networking/src/main/java/dto/DestinatieDTO.java
package dto;

import java.io.Serializable;

public class DestinatieDTO implements Serializable {

    private int id;
    private String nume;


    public Integer getId() {
        return id;
    }

    public void setId(Integer integer) {
        this.id=integer;
    }

    public DestinatieDTO(int id, String nume) {
        this.id = id;
        this.nume = nume;
    }

    public String getNume() {
        return nume;
    }

    public void setNume(String nume) {
        this.nume = nume;
    }

}

File: mpp/LabMppCurseWPF/Networking/DTOs/DestinatieDTO.cs
using System;

namespace Networking.DTOs
{	
	[Serializable]
	public class DestinatieDTO
	{
		private int _id;
		private string _nume;

		public DestinatieDTO(int id, string nume)
		{
			_id = id;
			_nume = nume;
		}

		public int Id
		{
			get { return _id; }
			set { _id = value; }
		}

		public string Nume
		{
			get { return _nume; }
			set { _nume = value; }
		}
	}
}